libs/utils/trace_analysis.py

```diff
@@ -951,46 +846,30 @@ def plotCPUFrequencyResidency(self, cpus=None):
                           loc='lower right')
             plt.subplots_adjust(hspace=0.5)
 
-    def getClusterFrequencyResidency(self, cluster):
+    def _getClusterActiveSignal(self, cluster):
         """
-        Get a DataFrame with per cluster frequency residency, i.e. amount of
-        time spent at a given frequency in each cluster.
+        Build a square wave representing the active (i.e. non-idle) cluster
+        time, i.e.:
+            cluster_active[t] = 1 if there is a task running in the cluster at
```

Comment:
That comment is misleading. We are looking at cpu_idle events, thus maybe better to say that:
```
   cluster_active[t] == 1 if the CPU is reported to be exited from an idle state by a cpu_idle event
```